test/unit/runnable.spec.js

```diff
@@ -109,11 +115,11 @@ describe('Runnable(title, fn)', function() {
 
   describe('#globals', function() {
     it('should allow for whitelisting globals', function(done) {
-      var test = new Runnable('foo', function() {});
-      assert(test.async === 0);
-      assert(test.sync === true);
-      test.globals(['foobar']);
-      test.run(done);
+      var runnable = new Runnable('foo', function() {});
+      assert(runnable.async === 0);
```

Comment:
Why have two properties (`sync` and `async`) representing a single state. And why aren't they both boolean?